web/src/tippyjs.ts

```diff
@@ -178,6 +178,25 @@ export function initialize(): void {
         placement: "right",
         delay: EXTRA_LONG_HOVER_DELAY,
         appendTo: () => document.body,
+        onShow(instance) {
+            const $container = $(instance.popper).find(".views-tooltip-container");
+            if ($container.data("view-code") === user_settings.web_home_view) {
+                $container.find(".views-tooltip-home-view-note").removeClass("hide");
+            }
```

Comment:
Have you checked whether all the existing `tippy-left-sidebar-tooltip` elements have a `views-tooltip-container` inside them? I don't see why this wouldn't break something, unless you're relying on `$container.data("view-code")` being `undefined` if the selector doesn't find anything, which feels like sloppy code.

Reply:
So I checked, all `tippy-left-sidebar-tooltip` is supposed to have `views-tooltip-container` with the refactor but went into the second commit. So `$container.data("view-code")` shouldn't return `undefined`.